pkg/generate/generate.go

```diff
@@ -221,8 +221,8 @@ func (c *Controller) applyGeneratePolicy(log logr.Logger, policyContext *engine.
 		}
 
 		// add configmap json data to context
-		if err := engine.LoadContext(log, rule.Context, resCache, policyContext); err != nil {
-			log.Error(err, "cannot add configmaps to context")
+		if err := engine.LoadContext(log, rule.Context, resCache, policyContext, rule.Name); err != nil {
+			log.Info("cannot add configmaps to context", "reason", err.Error())
```

Comment:
Why changing it to ingo log?

Reply:
Not sure. 
Reverting it back to `log.Error`.